documentation/docs/getting_started/docker.md

```diff
@@ -4,20 +4,117 @@ keywords:
 - docker
 ---
 
-# Docker
+# Setting up `Hornet` with `Chronicle` as INX plugin for permanent data storage and Tangle and Ledger live analytics
 
-## Usage
+This is a guide to set up a `Hornet` node together with `Chronicle` as an INX plugin to persist Tangle data without pruning (make sure to maintain sufficient hard disk storage), gather time series analytics data from the Tangle and the UTXO ledger, and display such data in meaningful ways using Grafana. 
 
-The easiest way to start Chronicle is by using our supplied Dockerfile.
+Once you've cloned the repository from [Github](https://github.com/iotaledger/inx-chronicle.git), switch to the `docker` folder:
+```sh
+cd inx-chronicle/docker
+```
+
+## 1. Create the necessary directories for all `Hornet` and `Chronicle` databases
+
+```sh
+sudo ./create_dirs.sh
+```
+
+## 2. Configure `Hornet` by editing the `docker-compose.yml` file
+
+```sh
+<your-favorite-text-editor> docker-compose.yml
+```
+
+### Head to `Hornet`s CLI section:
+
+```yml
+hornet:
+    command:
+```
+
+### (Optional) Select another network (default=Shimmer) by uncommenting one of the `config*.json` lines as shown below:
+
+```yml
+- "-c"
+# - "config_testnet.json"
+# - "config_alphanet.json"
+```
+### Configure peers for Hornet:
```

Comment:
Why are we including hornet docs in our docs? We should just refer to what they have right?

Reply:
I want this guide to be as complete as possible. You know how I love docs where you have to jump to x other places in order to get it everything together to make it worke? Right ... I don't. I could just refer to Hornet's docs, but there the context of using it with Chronicle is totally lost, and the reader probably just gets basic information of how to set up peering, possibly prefering of using a the `peering.json`, which would be more troublesome for the user to get working.

Reply:
But users SHOULD be setting up hornet independently of chronicle. Chronicle depends on hornet, not the other way around.

Reply:
but in this case Hornet is not set up independently because it's part of the docker environment.